tests/trainer/test_trainer.py

```diff
@@ -644,3 +645,49 @@ def test_default_args(tmpdir):
 
     assert isinstance(trainer, Trainer)
     assert trainer.max_epochs == 5
+
+
+@pytest.mark.parametrize(
```

Comment:
it starts to be quite long, may consider to move it to a new file like `trainer_cli` along with the existing args testing

Reply:
done